### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by Aleortela**
Added Instructor controller plus implemented create, delete, and update methods. It also includes the JUnit Test for the add assignment path.

### CODE REVIEW

File: cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java [hunk 1]
```diff
@@ -10,11 +11,11 @@ public class AssignmentListDTO {
 	public static class AssignmentDTO {
```
Comment (Bentheburrito) [inline cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java:11]: Should this class be `AssignmentListDTO`?

**Review by Bentheburrito — APPROVED** [review]
Just one question, but otherwise LGTM!

File: cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java (hunk 1 above)
Comment (Bentheburrito) [reply cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java:11]: Oops never mind, `public class AssignmentListDTO {` was hidden and I just had to expand it :sweat_smile:

Comment (Aleortela) [reply cst438_gradebook/src/main/java/com/cst438/domain/AssignmentListDTO.java:11]: Awesome, glad it wasn't anything I had to fix. Thank you!

**Pull request merged**